vizro-core/src/vizro/managers/_data_manager.py

```diff
@@ -63,11 +63,10 @@ def memoize(wrapped: BoundMethod, instance: _DynamicData, args, kwargs):
     Args:
         wrapped: bound method to memoize.
         instance: _DynamicData object to which wrapped is bound.
-        args: positional arguments supplied to wrapped, taken into account for generating cache key.
-        kwargs: keyword arguments supplied to wrapped, taken into account for generating cache key.
+        args : positional arguments supplied to wrapped, taken into account for generating cache key.
+        kwargs : keyword arguments supplied to wrapped, taken into account for generating cache key.
```

Comment:
Is that according to the google style for docstrings

Reply:
@maxschulz-COL It's necessary because otherwise `griffe` reports a warning and, because of `--strict` that translates to an error. I don't know why @antonymilne didn't have this when he merged though 🤷

Reply:
Strange... given that we do not have automatic API docs for the data manager, I would then just leave as is and not fuss anymore!

Reply:
That's the mystery solved! It was fussing because I accidentally re-included the page that pulls it in (@huong-li-nguyen spotted that my merge reintroduced it) so I fixed it, but it wasn't actually a problem for @antonymilne because it's been removed as you say. Thanks for clearing that up. Let's leave the fix in place for now though.